### ISSUE TITLE
on_after_filter triggers twice

### ISSUE THREAD

**Issue by KGFTW** [koalyptus/TableFilter#246]
Hi,
I've been using tablefilter for a while now and it's really awesome but I encountered an issue lately: the code defined in the on_after_filter event handler gets called twice.

The function called there is not referenced elsewhere and I didn't set the on_after_filter event handler elsewhere than in the config before to init the tf variable.

Do you have any idea where this might come from ?

**Edit :** 
Investigated a bit further, it seems the problem only occurs in Chrome.
Firefox is not affected.
Still don't get why.

**Comment by KGFTW** [comment]
Investigated a bit further, it seems the problem only occurs on Chrome.
Firefox is not affected.
Still don't get why.

**Comment by koalyptus** [comment]
Hi, didn't get a chance to look at this yet. Are you using persistence ? 
Can you post your config and version you are using.
Cheers

**Comment by KGFTW** [comment]
Hi Koalyptus,

Just updated to the latest version yesterday.
Here's my config : 
```
var filtersConfig = {
        base_path: "path/to/tableFilterFolder",
        col_0: 'none',
        col_1: 'checklist',
        col_2: 'checklist',
        col_3: 'checklist',
        col_4: 'checklist',
        col_5: 'none',
        col_6: 'none',
        col_7: 'none',
        popup_filters: true,
        mark_active_columns: true,
        on_after_filter:function() {
            console.log('After filter fired');
        }
    };
var tf = new TableFilter('myTableId', filtersConfig);
tf.init();
```
I haven't customized the filters nor their functions.
Acutally this happened a while ago but it did not really matter at that time, now I need it to only fire once for code execution purpose but don't get why it fires twice.. :(
I believe I don't use persistence though, my tf variable is reloaded and  every time the page is (TableFilter only used in one page).

**Comment by koalyptus** [comment]
Hi @KGFTW, 
I confirm the `on_after_filter` callback is fired twice when the filter is a `checklist` filter type. While waiting for a fix, an option could to use `multiple` filter type instead of `checklist` this would ensure the callback is fired once.

**Comment by koalyptus** [comment]
@KGFTW,
This should now be fixed in `v0.2.40`. Give it a try and tell me how it goes.
Cheers

**Comment by KGFTW** [comment]
@koalyptus Fixed, thanks a lot :)
